### ISSUE TITLE
jquery-ui css not inserted into application css

### ISSUE THREAD

**Issue by nicolasfranck** [DMPRoadmap/roadmap#2186]
Please complete the following fields as applicable:

**Expected behaviour:**

generate bundled application css

**Actual behaviour:**

the generated application.css contains css imports:
```
@import url(vendor/jquery-ui/datepicker/jquery-ui.min.css)
```
but these urls do not resolve.

apparently this is because the sass directive (of the same name)
generates css imports when the urls ends with .css

See https://sass-lang.com/documentation/at-rules/import

**Steps to reproduce:**

```
RAILS_ENV=production bundle exec rake assets:precompile
touch tmp/restart.txt
```

**Comment by nicolasfranck** [comment]
Proposal:

* in `app/assets/stylesheets/vendor/jquery-ui/datepicker`: move files `.min.css` to `.css` 
* in `app/assets/stylesheets/application.css`: remove postfix `.min.css`

Now the import is interpreted correctly, and the assets from datepicker are inserted instead of incorrectly referenced

**Comment by nicolasfranck** [comment]
this seems to be fixed in version 3.0.0, as I do not see this error anymore..not?
